lib/manager/gradle/index.js

```diff
@@ -0,0 +1,50 @@
+const { execSync } = require('child_process');
+const fs = require('fs');
+
+const gradle = require('./build-gradle');
+
+function extractDependencies(content, fileName, config) {
+  logger.debug('gradle.extractDependencies()');
+  const deps = [{ name: 'dummy' }];
+  const gradleFile = `${config.localDir}/${fileName}`;
+
+  fs.writeFileSync(gradleFile, content);
+  execSync('gradle dependencyUpdates -Drevision=release', {
+    stdio: 'inherit',
+    cwd: config.localDir,
+  });
+
+  return { deps };
+}
+
+function getPackageUpdates(config) {
```

Comment:
This function should probably be combined into the extractDependencies function

Reply:
Not sure if I understand this. Isn't this function part of the interface of the manager? I'm looking at the travis example. 

What I've understood is that this function returns a list of dependencies to be updated, and then the updateDependency function is called for each one of these

Reply:
I had forgotten about how Travis is implemented differently, and didn't realise you were copying from it. Yes, this can be a way to do this, so I'll take another look.